README.md

```diff
@@ -10,32 +10,51 @@ Ziggurat is written in stable Rust; you can install the Rust toolchain by follow
 
 ## Getting started
 
+### Preconditions
+
 1. Clone this repository.
 2. Build [go-algorand](https://github.com/algorand/go-algorand) from source.
 3. Run the setup script:
 ```zsh
-    tools/setup_env.sh
+ tools/setup_env.sh
 ```
-
 In case algorand files are installed in a specific location, export that location to the `ALGORAND_BIN_PATH`
 environment variable and rerun the setup script:
 ```zsh
-    export ALGORAND_BIN_PATH="$HOME/node"   # example path
-    tools/setup_env.sh
+ export ALGORAND_BIN_PATH="$HOME/node"   # example path
+ tools/setup_env.sh
+```
+
+### Run tests
+Run conformance and resistance tests with the following command:
+
+```zsh
+ cargo +stable test
+```
+
+### Run performance tests
+Create a package of IP addresses (4k addresses) which are required for performance tests. From the root repository directory run:
+
+#### Preconditions under Linux
+Generate dummy devices with addresses:
+```zsh
+ python3 ./tools/ips.py --subnet 1.1.0.0/20 --file src/tools/ips.rs --dev_prefix test_zeth
```

Comment:
If we remove `sudo` call from here and use it somewhere in the script, there should be notification here, that user must be in sudoers file to use this script.
Previous version of this script was privilages agnostic - it assumed that was run under root privilages (some Linux distro has no `sudo` by default and therefore user can use `su` to become root and run the script as usual - as user can see `sudo` in script invocation).

Reply:
Let me try to explain it in words in the form of a note before the section split (mac/Linux)

Reply:
Done